includes/wc-term-functions.php

```diff
@@ -25,11 +25,11 @@ function wc_get_object_terms( $object_id, $taxonomy, $field = null, $index_key =
 	// Test if terms exists. get_the_terms() return false when it finds no terms.
 	$terms = get_the_terms( $object_id, $taxonomy );
 
-	if ( ! $terms || is_wp_error( $terms ) || is_null( $field ) ) {
+	if ( ! $terms || is_wp_error( $terms ) ) {
 		return array();
 	}
 
-	return wp_list_pluck( $terms, $field, $index_key );
+	return is_null( $field ) ? $terms : wp_list_pluck( $terms, $field, $index_key );
```

Comment:
This changes will makes return different results now, suppose to return an empty string... Maybe include a new parameter or something else if need to return the list of terms even `$field` is `null`.

Reply:
That was the point :) Please see https://github.com/woocommerce/woocommerce/pull/20079

Reply:
@peterfabian ohh yeah, sorry.